js/dailyshine.js

```diff
@@ -141,6 +141,7 @@
         if (i == messages.length - 1) {
           data.linkTitle = localized(content.linkTitle);
           data.linkUrl = localized(content.linkUrl);
+          data.dateClicked = formatDateMMddyyy(new Date());
```

Comment:
Ahh, gotcha. For some reason I was thinking of the date when the user saw the Shine and clicked, as opposed to the date of the daily Shine, but it makes sense to capture the Shine’s date. Will fix this!